### PR TITLE
Added NorwegianList

### PR DESCRIPTION
**Pull request by Alex-302**
https://github.com/AdguardTeam/FiltersRegistry/issues/48

### CODE REVIEW

**Comment by MindTooth** [comment]
How does this stack with the old filter?  uBlock Origin removed it a long time ago, before the re-added this filter some versions ago.  If it supplements the old, or replace it all together.

**Comment by Alex-302** [comment]
It seems this filter is not dead.

@ameshkov can we merge it?

**Comment by MindTooth** [comment]
It's dead: http://home.fredfiber.no/langsholt/adblock.txt

**Comment by Alex-302** [comment]
Please look to commit:
https://raw.githubusercontent.com/DandelionSprout/adfilt/master/NorwegianList.txt

**Comment by ameshkov** [comment]
@MindTooth 
It's marked as "obsolete":
https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/ThirdParty/filter_120_NorskAdblockliste/metadata.json

We don't delete filters from this repo. Instead, we mark them "obsolete" and disable updates for such filters.

File: filters/ThirdParty/filter_249_NorwegianList/metadata.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "filterId": 249,
+  "name": "Dandelion Sprout's Norwegian Filters",
+  "description": " Dandelion Sprout's Norwegian Filters for tidier websites.",
+  "timeAdded": 1522412113779,
+  "homepage": "https://github.com/DandelionSprout/adfilt",
+  "expires": "2 days",
+  "displayNumber": 249,
+  "groupId": 3,
```
Comment (ameshkov) [inline filters/ThirdParty/filter_249_NorwegianList/metadata.json]: Wrong group, needs to be 7 -- language-specific

Comment (Alex-302) [reply filters/ThirdParty/filter_249_NorwegianList/metadata.json]: Done

**Pull request merged**